flags.go

```diff
@@ -74,19 +76,39 @@ func BuildFlags(obj any) Flags {
 	if v.Kind() != reflect.Struct {
 		panic(fmt.Errorf("expected a struct, got %s", v.Kind()))
 	}
+
+	return buildFlagsRecursive(v)
+}
+
+func buildFlagsRecursive(v reflect.Value) Flags {
 	t := v.Type()
+	var flags Flags
 
-	var err error
-	flags := make([]Flag, t.NumField())
 	for i := 0; i < t.NumField(); i++ {
-		flags[i], err = buildFlag(v.Field(i), t.Field(i))
-		if err != nil {
-			panic(err)
+		field := t.Field(i)
+		fieldValue := v.Field(i)
+
+		// Only process fields with a 'short' or 'long' tag
+		if hasTag(field.Tag, "short"); hasTag(field.Tag, "long") {
```

Comment:
I didn't know you can actually do this in Go.:)

Reply:
hahaha, my bad! changed the conditional and left it like that 🤦 . Fixing